apps/bash_shared/common.sh

```diff
@@ -1,7 +1,7 @@
 function registerHooks() { hwc_event_register_hooks "$@"; }
 function runHooks() { hwc_event_run_hooks "$@"; }
 
-source "$AC_PATH_CONF/config.sh.dist" # "hack" to avoid missing conf variables
+source "$AC_PATH_CONF/dist/config.sh" # include dist to avoid missing conf variables
```

Comment:
why renaming/moving the `config.sh.dist` ?

Reply:
for 2 reasons:

1) copy/past is easier, you don't have to change the name of the file

2) .dist file are not recognized by editors, you need to specify the syntax type manually.